### ISSUE TITLE
Cursor becomes stuck - j,k doesn't work

### ISSUE THREAD

**Issue by 3nuc** [asvetliakov/vscode-neovim#126]
![vscode stuck at bottom](https://user-images.githubusercontent.com/5359825/71534949-8ec26780-2902-11ea-8ad7-b0534aecd5fa.gif)

This issue happened immediately after starting VS Code. The editor groups and tabs were saved from a previous session. As you can see, the cursor is stuck at the bottom of a file and pressing j,k doesn't move it. I also clicked with the mouse to show that it's impossible to move the vim cursor at all.

The cursor can sometimes become stuck in the middle the file, but that may be a separate issue because it becomes unblocked after a while.

Maybe it's because the file `Two.vue` is opened in two editor groups (left and right - look at the tabs above) is causing the issue?

vscode 1.41.1 with some extensions enabled
nvim 0.4.3 with 0 plugins and empty init.vim
vscode-neovim 0.0.45
ubuntu 19.04

Similar to #85 ?

**Comment by asvetliakov** [comment]
No. It's just some strange race condition with nvim windows switching. Workaround for now should be move cursor to the other group, then move back (or switch tab, or close-reopen the editor)

**Comment by asvetliakov** [comment]
One reproducible case - open repo's `README.md` in the second column, sometimes need to close-reopen, sometimes pressing 'gg' from the middle of the file hangs

**Comment by virgil9306** [comment]
@asvetliakov I am still having this issue with latest versions.

* One buffer doesn't work, then I open different file in new buffer.
* This new buffer is OK, but the old one still doesn't respond when I use non-insert mode like h,j,k,l.

**Comment by asvetliakov** [comment]
Open new issue with repro steps and clean init.vim

**Comment by virgil9306** [comment]
After I specified a new, clean `init.vim` it is working fine, thanks @asvetliakov . My apologies.
